plugins/webp-uploads/settings.php

```diff
@@ -121,6 +133,23 @@ function webp_uploads_generate_webp_jpeg_setting_callback(): void {
 	<?php
 }
 
+/**
+ * Renders the settings field for the 'webp_uploads_use_picture_element' setting.
+ *
+ * @since 1.0.0
+ */
+function webp_uploads_use_picture_element_callback(): void {
```

Comment:
Something seems amiss here. 

`trunk` | Ths branch
--|--
![image](https://github.com/WordPress/performance/assets/134745/0af90cb2-8afb-473b-9988-9959de7cc83c) | ![image](https://github.com/WordPress/performance/assets/134745/587b2567-e315-4425-97b0-02ba57d72dc7)

I don't see the Picture section and Uploading Files section is made too narrow.

Reply:
👍🏼 on my list to fix that